ui/src/app/cohort-search/cohort-page/cohort-page.component.spec.tsx

```diff
@@ -11,9 +11,13 @@ import {CohortBuilderServiceStub} from 'testing/stubs/cohort-builder-service-stu
 import {CohortsApiStub} from 'testing/stubs/cohorts-api-stub';
 import {workspaceDataStub} from 'testing/stubs/workspaces';
 import {CohortPage} from './cohort-page.component';
-import {MemoryRouter} from "react-router";
+import {Router} from "react-router";
+import { createMemoryHistory } from 'history';
 
 describe('CohortPage', () => {
+  let props;
+  let history = createMemoryHistory();
```

Comment:
I would suggest as a pattern initializing this in `beforeEach` instead, to avoid undesired state bleeding across test cases.